packages/components/src/higher-order/with-constrained-tabbing/README.md

```diff
@@ -32,9 +31,7 @@ const MyComponentWithConstrainedTabbing = withState( {
 	}
 
 	const toggleConstrain = () => {
-		setState( ( state ) => ( {
-			isConstrainedTabbing: ! state.isConstrainedTabbing,
-		} ) );
+		setIsConstrainedTabbing( ( state ) => ! state );
```

Comment:
I guess `state` here would be better to be renamed to `isConstrainedTabbing`, no?

Reply:
Using `state` here to avoid the [no-shadow](https://eslint.org/docs/rules/no-shadow) ESLint rule.